### ISSUE TITLE
Update callout tour highlights to match design

### ISSUE THREAD

**Issue by bethanylang** [google/site-kit-wp#6973]
## Description

As reported by Sigal in Bug Bash, the highlights for specific sections of the dashboard do not match the design. For all three:

1. The highlighted white part is not in the correct size (too large) and placement
2. The highlighted white part should have a shadow like the tooltip instead of the green highlight it currently has

Note that this is blocked by #6904 and #7001, where we need to update the position of the New badges before modifying the highlights.

---------------

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Acceptance criteria

- The target highlight of the GA4 Reporting feature tour steps should match [the Figma design](https://www.figma.com/file/0UCIE3hPeiu4NYdpAODASd/GA4-Updates?node-id=1-2598&t=bRMnSweXXrOVqOHP-0) in terms of size and placement. The highlight should have a consistent padding around the target element according to [the Figma design](https://www.figma.com/file/0UCIE3hPeiu4NYdpAODASd/GA4-Updates?node-id=1-2598&t=bRMnSweXXrOVqOHP-0), and the target element should be centered inside the highlight.
- The green outline should be removed from the highlight so that it matches [the Figma design](https://www.figma.com/file/0UCIE3hPeiu4NYdpAODASd/GA4-Updates?node-id=1-2598&t=bRMnSweXXrOVqOHP-0).

## Implementation Brief

- In `assets/js/feature-tours/ga4-reporting.js`:
  - Add a `styles` key to each item in the `steps` array. The value of `styles` should be an object having a `spotlight` key, with the value being another object with the following keys:
    - `border` with the value of `0`.
    - `padding` with the value of 15px in the bottom (e.g. `0 0 15px 0`).

### Test Coverage

- No tests need to be added/updated.

## QA Brief

* Load the GA4 Feature Tour by having a site with the `ga4Reporting` feature flag enabled, connecting Analytics, and ensuring the dashboard view used is the GA4 dashboard view.
* Go to the main dashboard and trigger the feature tour by clicking on the "Learn what's new" CTA button in this notification:

![CleanShot 2023-05-15 at 02 19 56](https://github.com/google/site-kit-wp/assets/90871/d01f7ab3-9108-4b71-af1e-ee94b6aa323c)

* The feature tour spotlight should have the correct padding around the elements highlighted; see the Figma links in the issue for reference.

----

* This issue also prevents the feature tour from running on mobile viewports. If you follow the QA steps here on mobile, you should see a notification with only an "OK, Got it!" button that dismisses the notification temporarily but does not mark the tour as complete, meaning if you type `sessionStorage.clear()` in the browser JS console and refresh the page, you will see the notification again and can run the tour once switched to a desktop viewport.

## Changelog entry

* Update GA4 Reporting feature tour highlights to match design.

**Comment by eugene-manuilov** [comment]
AC ✔️

**Comment by wpdarren** [comment]
I am adding an observation that I discovered while testing #6904.

Now that we have made the changes to the badge placement, we've created a UI issue with the tooltip tour.  As you can see from the screenshots below, the tooltip slightly overlaps the text.  Is it possible to fix this within this ticket, or, do we have another ticket that it can be fixed in?  I can create a new ticket, just conscious this would likely be a new P0. 

![image](https://user-images.githubusercontent.com/73545194/235893732-1f2b32a7-fed9-411b-810c-4fdcf6e6c85d.png)
![image](https://user-images.githubusercontent.com/73545194/235893778-5c40dd06-8a03-4f36-bf1e-0e63a1c6b91c.png)

**Comment by techanvil** [comment]
@wpdarren I think we're OK addressing this here - the first point of the AC should cover it:

> - The target highlight of the GA4 Reporting feature tour steps should match [the Figma design](https://www.figma.com/file/0UCIE3hPeiu4NYdpAODASd/GA4-Updates?node-id=1-2598&t=bRMnSweXXrOVqOHP-0) in terms of size and placement.

Note that the badges are similarly aligned in the Figma design linked to:

![image](https://user-images.githubusercontent.com/18395600/235952850-71774bf1-5c35-4e7d-b7bf-5885290837ad.png)

![image](https://user-images.githubusercontent.com/18395600/235953037-46bdd0bf-e70b-432d-b64a-36c08cbc27af.png)

**Comment by eugene-manuilov** [comment]
IB ✔️

**Comment by mohitwp** [comment]
### QA Update ❌ 

@tofumatt 

- Tested on dev environment.
- Tested tour highlights in desktop and tablet view ports.
- Verified in mobile view ports "Learn what's new' button is not showing and also tour is not getting trigger.
- Verified The highlight should have a consistent padding around the target element according to [the Figma design](https://www.figma.com/file/0UCIE3hPeiu4NYdpAODASd/GA4-Updates?node-id=1-2598&t=bRMnSweXXrOVqOHP-0), and the target element is now centered inside the highlight.
- Verified the green outline is now removed from the highlight.

**Issue** :- I've noticed that in tablet view ports having width between 783px to 1179px, the session target highlight is overlapping the engagement title. 

<details>

![image](https://github.com/google/site-kit-wp/assets/94359491/b5b459c2-1382-4dd0-adac-0bbed7c14105)

https://github.com/google/site-kit-wp/assets/94359491/5ddc4bc9-0f5f-4226-b5ed-32f9e8266850

</details>

**Question** – In IB it is mentioned that top padding is 0px and bottom padding is 15px. But under actual implementation top padding is 12px and bottom padding is 20px which looks more correct. Did we implement different padding from the suggested padding in IB so that target element gets centered around the highlight ?

<details>

![image](https://github.com/google/site-kit-wp/assets/94359491/4387aeba-a330-4309-9697-6ab5231760d9)

![image](https://github.com/google/site-kit-wp/assets/94359491/e0c4739d-0c84-4d81-afb9-e22c21c73198)

![image](https://github.com/google/site-kit-wp/assets/94359491/2f177fff-1188-4683-8a80-8333e0ad1b96)

</details>

**Comment by tofumatt** [comment]
@mohitwp Yes, the padding is different to look better; it's always best to compare the ACs not the IB, as sometimes we deviate a bit 😄 

Thanks for catching the tablet padding, I'll get a follow-up PR for that one 👍🏻

**Comment by tofumatt** [comment]
@mohitwp I just spent a fair bit of time on this, trying to get the widths to align perfectly on smaller screens, but because of React Joyride's own logic around spotlights and those headers being VERY small, it doesn't seem to work.

For now I think the spotlight being wider than the contents on smaller screens will need to be a known bug, so moving this to Approval as the QA was otherwise good and I think that width on tablets is a "Won't fix" issue, at least in the context of this issue.

**Comment by aaemnnosttv** [comment]
LGTM!
